FiveCalls/FiveCalls/IssueDetailViewController.swift

```diff
@@ -10,6 +10,7 @@ import UIKit
 import CoreLocation
 import Crashlytics
 import Down
+import Mixpanel
```

Comment:
don’t need this in your view controllers anymore, since you’re calling an extension method.